examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
```

Comment:
oh yeah... i just picked some random hashes. I can make them real, but then we have to worry about pinning them forever.